projects/packages/sync/src/modules/class-users.php

```diff
@@ -31,6 +31,23 @@ class Users extends Module {
 	 */
 	protected $flags = array();
 
+	/**
+	 * Mapping between user fields to flags.
+	 *
+	 * @var array
+	 */
+	protected $user_fields_to_flags_mapping = array(
+		'user_pass'           => 'password_changed',
+		'user_email'          => 'email_changed',
+		'user_login'          => 'login_changed',
+		'user_nicename'       => 'nicename_changed',
+		'user_url'            => 'url_changed',
+		'user_registered'     => 'registration_date_changed',
+		'user_activation_key' => 'activation_key_changed',
+		'user_status'         => 'status_changed',
+		'display_name'        => 'display_name_changed',
```

Comment:
Will these flags also mean the message is correct too? e.g. 'User display name changed', instead of 'User modified'?

Reply:
Yes, this is the final goal! If no one objects to this change in the linked P2, I plan to create a follow up diff on WPCOM that will take care of this.

Reply:
🙌